castle/cms/profiles/default/registry/controlpanel.xml

```diff
@@ -120,16 +118,4 @@
     </value>
   </record>
 
-  <record name="castle.es_index"
-          interface="castle.cms.interfaces.IElasticSearchSettings"
-          field="es_index">
-    <field type="plone.registry.field.TextLine" />
-  </record>
-
-  <record name="castle.es_index_enabled"
-          interface="castle.cms.interfaces.IElasticSearchSettings"
-          field="es_index_enabled">
-    <field type="plone.registry.field.Bool" />
-  </record>
-
 </registry>
```

Comment:
No, you're right in that those would be expected. I did not add them though because I wanted to ensure the update wasn't destructive (in case of rollback, or some need of previously configured values).

They do leave around a few registry settings that will be functionally useless, but they also leave around a path for the previous version to be re-applied if needed.

Might be good to add a note in the history file about them staying around and that they could be manually removed if needed.

Reply:
included note in history in next commit